readme_collector.py

```diff
@@ -0,0 +1,59 @@
+from coc_collector import *
+
+class Readme_Statuses(Enum):
+    """
+    This class enumerates the various potential states of a repository's
+    code of conduct.
+    """
+    GIT_DET_NONE = 1
```

Comment:
this seems like a nit.. but can we make these "GH_DETERMINED_NONE" and "GH_DETERMINED_YES"? cause a) it is *github* not *git* making the determination and b) "det" is not normally short for "determined"